forte/data/entry_type_generator.py

```diff
@@ -39,3 +49,25 @@ def get_type_attributes():
 
         """
         return _get_type_attributes()
+
+    @staticmethod
+    def get_entry_attribute_by_class(input_entry_class: str):
```

Comment:
1. missing return type
2. wrong documentation format, missing arguments, missing return types.  
3. To be more accurate, we should probably write the docstring with [doctest format](https://docs.python.org/3/library/doctest.html) (which will ensure the documentation will also be tested!) , i.e. using `>>>` and return values. We often use doctest for utilities, like [here](https://github.com/asyml/forte/blob/master/forte/utils/utils.py#L240).

Reply:
Modified in new PR.